fury/ui/containers.py

```diff
@@ -65,29 +68,35 @@ def _setup(self):
         self.element_offsets = []
         self.background = Rectangle2D()
 
-        self.borders = {'left': Rectangle2D(),
-                        'right': Rectangle2D(),
-                        'top': Rectangle2D(),
-                        'bottom': Rectangle2D()}
+        if self.has_border:
+            self.borders = {'left': Rectangle2D(),
+                            'right': Rectangle2D(),
+                            'top': Rectangle2D(),
+                            'bottom': Rectangle2D()}
 
-        self.border_coords = {'left': (0, 0),
-                              'right': (0, 0),
-                              'top': (0, 0),
-                              'bottom': (0, 0)}
+            self.border_coords = {'left': (0., 0.),
+                                  'right': (1., 0.),
+                                  'top': (0., 1.),
+                                  'bottom': (0., 0.)}
+        else:
```

Comment:
`else` not needed, can you just move the initialization above `if self.has_border`

Reply:
Done with [c43adff](https://github.com/fury-gl/fury/pull/441/commits/c43adff5b8f7f21d539bf58dc8e2f12d4d0cf640).